pinot-transport/src/main/java/com/linkedin/pinot/routing/HelixExternalViewBasedRouting.java

```diff
@@ -137,6 +137,12 @@ public HelixExternalViewBasedRouting(ZkHelixPropertyStore<ZNRecord> propertyStor
     return serverToSegmentSetMaps.get(_random.nextInt(serverToSegmentSetMaps.size())).getRouting();
   }
 
+  @Override
```

Comment:
According to the if statements in line 106 and 107, it seems that we route to llc ONLY if the table is a REALTIME table AND it does not have any entries in the HLC part. So, i am not clear on your comment that we bump the llc routing metric even if the table is offline only. Is there something else going on that we are missing?